### PR TITLE
Feedback service included on all pages

### PR DESCRIPTION
**Pull request by ulrichenslin**

### CODE REVIEW

File: admin/contribution/index.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ permalink: admin/contribution/
 {% if article.path contains context %}
 {% assign found_content = true %}
 {{ article.content }}
+{% include feedback.html %}
```
Comment (gauravlall) [inline admin/contribution/index.md]: This document references contributions in developer section, I don't think there is a need for feedback here.

File: user/contribution/index.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ permalink: user/contribution/
 {% if article.path contains context %}
 {% assign found_content = true %}
 {{ article.content }}
+{% include feedback.html %}
```
Comment (gauravlall) [inline user/contribution/index.md]: This document references contributions in developer section, I don't think there is a need for feedback here.

File: admin/contribution/index.md (hunk 1 above)
Comment (ulrichenslin) [inline admin/contribution/index.md]: Removed from contribution sections

File: user/contribution/index.md (hunk 2 above)
Comment (ulrichenslin) [inline user/contribution/index.md]: Removed from contribution sections

**Pull request merged**